cdk/domino_cdk/provisioners/s3.py

```diff
@@ -72,23 +61,5 @@ def provision_buckets(self, name: str, s3: List[config.S3]):
                     conditions={"Null": {"s3:x-amz-server-side-encryption": "true"}},
                 )
             )
-            self.s3_api_statement.add_resources(f"{self.buckets[bucket].bucket_arn}*")
-            cdk.CfnOutput(self.parent, f"{bucket}-output", value=self.buckets[bucket].bucket_name)
 
-    def provision_iam_policy(self, name: str):
-        self.policy = iam.ManagedPolicy(
-            self.scope,
-            "S3",
-            managed_policy_name=f"{name}-S3",
-            statements=[
-                iam.PolicyStatement(
-                    actions=[
-                        "s3:ListBucket",
-                        "s3:GetBucketLocation",
-                        "s3:ListBucketMultipartUploads",
```

Comment:
`ListBucket` isn't present in the new permissions at all, and the other two here are now gated to the bucket. I believe (though don't know for a fact) that these permissions were here and ungated for a reason. That reason may be down to how Nucleus or some other service accesses the buckets. We can leave it how it is and see how things go, but at the very least thought I should mention it.

Reply:
Yes, I was copying these permission from somewhere else, not from the node permissions.

ListBucket cannot be bound to a single bucket because it lists al buckets.